Plugins/Administration/Administration.cpp

```diff
@@ -122,6 +154,7 @@ Events::ArgumentStack Administration::DeletePlayerCharacter(Events::ArgumentStac
 {
     const auto objectId = Events::ExtractArgument<ObjectID>(args);
     const auto bPreserveBackup = static_cast<bool>(Events::ExtractArgument<int32_t>(args));
+    const auto retainTurd = Events::ExtractArgument<int32_t>(args);
```

Comment:
Should probably wrap this in a try/catch for a little while:

```c
int32_t retainTurd;
// TODO: Remove this try/catch at some later point
try
{
    retainTurd  = Services::Events::ExtractArgument<int32_t>(args);
}
catch (const std::runtime_error& e)
{
    retainTurd  = false;
}
```

Reply:
Decided I was being too fancy allowing for TURD preservation and just nixed that logic.